english/zoo/zoo.md

```diff
@@ -1,2 +1,5 @@
 Recently I visited a zoo in my hometown. It was an awesome experience to be part of the
 nature. I saw a lot of tigers, elephants, deer and a whole lot of other [animals](../animal/animal.md)
+
+I was pretty thirsty. There is a lemonade stand over there.
+I head over and buy some [lemonade](../drinks/lemonade.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.